### PR TITLE
Upgraded traptor.py to python 3

### PR DESCRIPTION
**Pull request by pklene96**
# For the Reviewer
- [ ] Code review complete
- [ ] Testing Complete
- [ ] Quality ORT App Documentation Updated (your name is in the Validator square for this feature)

When this is complete, you should approve the PR via github.

# For the Reviewee

<Reminder PR Title should be JIRA_NUMBER and Useful Description>

## Summary
Description of changes.

#### Release Note
<Concise sentence describing change>Required.

#### Breaking Changes
<Description for Techops of how to handle changes, migrations, updates>None.

## Quality Assurance

You have gathered the following items:
- [ ] This PR is tagged with a Release Milestone
- [ ] You have a log message clearly identifying when this feature is **working successfully**
- [ ] You have a log message clearly identifying when this feature is **failing**
- [ ] You added a PR against [p4-alerting](https://github.com/istresearch/p4-alerting/) to trigger based on the failure condition above

Given all of the items above, you have updated your Application ORT at the following locations:
- **Features and Alerting**: <link to app ORT>Required.
- **P4 Alerting**: <link to p4-alerting PR>Required.

## Testing and Verification

*Steps to test your application for someone not familiar with it.* Required.

1. Configure Kafka without security to ensure this still works. Edit the kafka docker-compose service to make it available outside the docker network, as shown below.
```
  kafka:
    image: wurstmeister/kafka
    ports:
      - "9092:9092"
    environment:
      KAFKA_LISTENERS: PLAINTEXT://:9092
      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://192.168.1.198:9092   <---- Your IP here.
      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
      KAFKA_CREATE_TOPICS: "traptor:1:1"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
    depends_on:
      - zookeeper
```
2. Start Kafka and zookeeper 
```
$ docker-compose up -d zookeeper kafka
```
3. Use secure pipes to connect to redis.

4. Configure the Traptor settings for the Kafka writer in configs/settings.py.
```
KAFKA_HOSTS = '192.168.1.198'  #                                          <---- Use your machine IP here
KAFKA_BROKER_VERSION_FALLBACK = '0.9.0'
KAFKA_API_VERSION_REQUEST = True
...
KAFKA_SECURITY_PROTOCOL = None  #                               <---- Use None
...
```
5. Run the traptor.py file and look for confirmation a document was written.

6. Confirm data on the topic using a console consumer. Be sure to specify your machine IP in place of 192.168.1.198 in the command that follows.
```
$ docker run -it --network=host edenhill/kafkacat:1.6.0 -b 192.168.1.198:9092 -t traptor
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -65,12 +65,12 @@ services:
...
   kafka:
-    image: wurstmeister/kafka:0.9.0.1-1
+    image: wurstmeister/kafka
     ports:
       - "9092:9092"
     environment:
-      #KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092    # If trying to connect from outside docker, enable this
-      #KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092    # If trying to connect from inside docker, enable this
+      KAFKA_LISTENERS: PLAINTEXT://:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://192.168.1.198:9092
```
Comment (rsdunya) [inline docker-compose.yml:73]: Should we revert lines 72 and 73 back to the master config?

File: traptor/settings.py [hunk 2]
```diff
@@ -27,9 +27,17 @@
 
 # Kafka
 KAFKA_ENABLED = 'True'
-KAFKA_HOSTS = "localhost:9092"                  # Kafka host(s)
 KAFKA_TOPIC = "traptor"                         # Kafka topic to write all twitter data
 
+KAFKA_BOOTSTRAP_SERVERS = 'localhost:9092'
+KAFKA_HOSTS = '127.0.0.1'
```
Comment (rsdunya) [inline traptor/settings.py:33]: Should this be:
KAFKA_HOSTS = "localhost:9092" ?

File: docker-compose.yml (hunk 1 above)
Comment (pklene96) [reply docker-compose.yml:73]: Yes!

**Review by rsdunya — APPROVED** [review]
Met with @pklene96 to discuss and verify test steps. Minor change requests requested and completed.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.7'
```
Comment (andrewkcarter) [inline docker-compose.yml:1]: This is a docker-compose format version, not python. Need to revert that back to '2'
```suggestion
version: '2'
```

**Review by andrewkcarter — APPROVED** [review]

**Pull request merged**